### ISSUE TITLE
Opt-in for email notification

### ISSUE THREAD

**Issue by 2403905** [owncloud/ocis#7762]
# Description
For [redacted] we want to be able to disable email notifications by default. Users still should be able to opt-in to email notifications.

[https://github.com/owncloud/enterprise/issues/5932](https://github.com/owncloud/enterprise/issues/5932)

## User Stories

* > As a ..., I want to ... so that ... (please stick to who, what, why)

## Definition of ready
- [ ] Everybody needs to understand the value written in the user story
- [ ] Acceptance criteria have to be defined
- [ ] All dependencies of the user story need to be identified
- [ ] Feature should be seen from an end user perspective
- [ ] Story has to be estimated
- [ ] Story points need to be less than 20

## Definition of done
- Functional requirements
  - [ ] Functionality described in the user story works
  - [ ] Acceptance criteria are fulfilled
- Quality
  - [ ] Code review happened
  - [ ] CI is green (that includes new and existing automated tests)
  - [ ] Critical code received unit tests by the developer
- Non-functional requirements
  - [ ] No sonar cloud issues
- Configuration changes
  - [ ] The next branch of the ocis charts is compatible

**Comment by tbsbdr** [comment]
No, not from my side (though it should be someday configurable)
